app/src/main/java/com/xa/xpensauditor/MonthWiseVis.java

```diff
@@ -47,8 +50,9 @@ public class MonthWiseVis extends AppCompatActivity {
     private Button Vis;
     private String Tid;
     int tot=0;
-
+    int ii=0;
     static Map<Integer, Integer> hm = new HashMap<>();
+    Map<Date, Integer> sortedMap = new TreeMap<Date, Integer>();
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'member def type' has incorrect indentation level 4, expected level should be 2.